api/security.go

```diff
@@ -530,3 +531,14 @@ func GetReleaseScanLog(c *gin.Context) (map[string]bool, bool) {
 	log.Debugf("ReleaseScanLogReject set: %#v", releaseScanLogReject)
 	return releaseScanLogReject, true
 }
+
+// SecurytiScanEnabled checks if security scan is enabled in pipeline
+func SecurytiScanEnabled(c *gin.Context) {
+
+	if viper.GetBool("anchore.enabled") {
+		c.Status(http.StatusOK)
+		return
+	}
+	c.Status(http.StatusBadRequest)
```

Comment:
I would return some other 200 always and the body should contain the boolean I guess, because this is not a client error.